client/default_keys.dic

```diff
@@ -944,5 +944,9 @@ A23456789123
 A00003000084
 675A32413770
 395244733978
+675A32413770
+395244733978
```

Comment:
isn't these two duplicates from above?

Reply:
Yep, totally messed up the repos and commit here : D
Missing keys are:
4243414F5250
DFE73BE48AC6